modules/custom/cu_photo_gallery_bundle/cu_photo_gallery_bundle.install

```diff
@@ -7,5 +7,6 @@ function cu_photo_gallery_bundle_enable() {
 
 }
 
-
-
+function cu_photo_gallery_bundle_update_7001() {
```

Comment:
We only want to run secure_permissions_rebuild once in cu_core AND that isn't even necessary for this release since SE and SO are synched